internal/index_read.go

```diff
@@ -103,14 +102,15 @@ func readBins(r io.Reader, typ string) ([]Bin, *ReferenceStats, error) {
 	return bins, stats, nil
 }
 
-func readChunks(r io.Reader, n int32, typ string, buf [16]byte) ([]bgzf.Chunk, error) {
+func readChunks(r io.Reader, n int32, typ string) ([]bgzf.Chunk, error) {
 	if n == 0 {
 		return nil, nil
 	}
 	chunks := make([]bgzf.Chunk, n)
+	buf := make([]byte, 16)
```

Comment:
No, I mean on this line, use `var buf [16]byte` and then pass `buf[:]` to `io.ReadFull` below. Does that make a difference - it shouldn't, but that is easier to read.